app/src/main/java/waiter/Router.java

```diff
@@ -0,0 +1,30 @@
+package waiter;
+
+import java.util.Map;
+
+public class Router {
+
+    Map<String, Map<String, String>> routes;
+
+    public Router() {
+        this.routes = Map.ofEntries(
+                Map.entry("GET /simple_get", Map.ofEntries(
+                        Map.entry("status", "200 OK"),
+                        Map.entry("headers", "Accept-Language: en-US\r\nContent-Length: 0"),
+                        Map.entry("body", "")
+                )),
+                Map.entry("GET /simple_get_with_body", Map.ofEntries(
+                        Map.entry("status", "200 OK"),
+                        Map.entry("headers", "Accept-Language: en-US\r\nContent-Length: 11"),
+                        Map.entry("body", "Hello world")
```

Comment:
I'm starting to see some patterns emerge here for setting data on the response.

Also, the `Content-Length` value could be calculated.

Reply:
Should I handle this pattern differently? Will try to calculate the content length 😄 thank you!

Reply:
I think this pattern is fine for now, but I would consider how to extract some of the repetition as you continue to build this out.

Reply:
Will do, Ill look into it!

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/14/commits/3e780f258f08e75533484a8eec8eec26e229510b), Since it made it easier to calculate the content length, I did a little extraction on the routes construction too, let me know if anything looks worse! Thank you 😄